components/org.wso2.carbon.identity.branding.preference.management.core/src/main/java/org/wso2/carbon/identity/branding/preference/management/core/constant/BrandingPreferenceMgtConstants.java

```diff
@@ -66,8 +68,25 @@ public enum ErrorMessages {
                 "Unsupported Encoding in the branding preference configurations of the tenant: %s."),
         ERROR_CODE_NOT_ALLOWED_BRANDING_PREFERENCE("BRANDINGM_00011",
                 "Requested branding preference configuration: %s is not allowed for the organization."),
-        ERROR_CODE_ERROR_VALIDATING_BRANDING_PREFERENCE("BRANDINGM_00012",
-                "Error while validating branding preference configurations for the organization: %s.");
+        ERROR_CODE_ERROR_VALIDATING_BRANDING_PREFERENCE("BRANDINGM_00021",
+                "Error while validating branding preference configurations for the organization: %s."),
+        // Error messages related to custom text configurations.
+        ERROR_CODE_INVALID_CUSTOM_TEXT_PREFERENCE("BRANDINGM_000022",
+                "Invalid custom text configurations for tenant: %s."),
+        ERROR_CODE_CUSTOM_TEXT_PREFERENCE_NOT_EXISTS("BRANDINGM_000023",
+                "Custom Text preferences are not configured for tenant: %s."),
+        ERROR_CODE_CUSTOM_TEXT_ALREADY_EXISTS("BRANDINGM_00024",
+                "Custom Text preference already exists for tenant: %s."),
+        ERROR_CODE_ERROR_GETTING_CUSTOM_TEXT_PREFERENCE("BRANDINGM_00025",
+                "Error while getting branding preference configurations for tenant: %s."),
+        ERROR_CODE_ERROR_ADDING_CUSTOM_TEXT_PREFERENCE("BRANDINGM_000026",
+                "Unable to add branding preference configurations for tenant: %s."),
+        ERROR_CODE_ERROR_DELETING_CUSTOM_TEXT_PREFERENCE("BRANDINGM_000027",
+                "Unable to delete branding preference configurations for tenant: %s."),
+        ERROR_CODE_ERROR_UPDATING_CUSTOM_TEXT_PREFERENCE("BRANDINGM_000028",
+                "Unable to update branding preference configurations."),
+        ERROR_CODE_ERROR_BUILDING_CUSTOM_TEXT_PREFERENCE("BRANDINGM_00029",
```

Comment:
Some error codes have 4 zeros, but it should be 3 zeros as per the used format in previous error codes